docs/docs/10-users/20-getting-started/40-connect-code.md

```diff
@@ -16,9 +16,10 @@ In this case, we already provide you with the code here:
 
 :::info
 
-You can connect code repositories on GitHub and GitLab.com, too. Check out the **Integrations** page (accessible via the 👤 user menu in the top right) to connect your accounts and sync access to repositories. For more information, see [How to connect your Renku account to your GitHub or GitLab account](../code/guides/connect-renku-account-to-github-or-gitlab-account).
-And, if your institution has a dedicated GitLab instance you would like to integrate with Renku, do not hesitate to [Contact](../community)  us, and we will be happy to add it to the list.
+You can connect code repositories from GitHub and [GitLab.com](https://about.gitlab.com/) to RenkuLab, too! Check out the **Integrations** page (accessible via the 👤 user menu in the top right) to connect your accounts and sync access to repositories. For more information, see [How to connect your Renku account to your GitHub or GitLab account](https://docs.renkulab.io/en/latest/docs/users/code/guides/connect-renku-account-to-github-or-gitlab-account). 
```

Comment:
* Separate into paragraphs for easier reading
* Add a link to GitHub (not just GitLab)
* Fix link to Renku/GitHub-GitLab integration page

```suggestion
You can connect code repositories from [GitHub](https://github.com) and [GitLab.com](https://about.gitlab.com/) to RenkuLab, too! 

Check out the **Integrations** page (accessible via the 👤 user menu in the top right) to connect your accounts and sync access to repositories. For more information, see [How to connect your Renku account to your GitHub or GitLab account]((../50-code/00-code-repository.md). 
```